### PR TITLE
copyrightholder added into metadata SDPA-268

### PR DESCRIPTION
**Pull request by vied12**
requires https://github.com/superdesk/superdesk-core/pull/229

### CODE REVIEW

File: scripts/superdesk-archive/views/upload.html [hunk 1]
```diff
@@ -43,6 +43,11 @@ <h3 class="pull-left">{{ activity.label|translate }}</h3>
                         <div class="row"><input type="text" placeholder="{{ :: 'Add a slugline' | translate }} *" ng-model="item.meta.slugline"></div>
                         <div class="row"><input type="text" placeholder="{{ :: 'Add author / photographer' | translate }}" ng-model="item.meta.byline"></div>
                         <div class="row">
+                            <input type="text"
+                                   placeholder="{{ :: 'Who is claiming the property' | translate }}"
```
Comment (ahilles107) [inline scripts/superdesk-archive/views/upload.html]: it can't be one-liner (like other inputs?)

**Comment by ahilles107** [comment]
Where i can find info why `usageterms` are removed? Would be ok write few lines of description in PR about that change. As it's also not mentioned in title.

File: scripts/superdesk-archive/views/upload.html (hunk 1 above)
Comment (vied12) [inline scripts/superdesk-archive/views/upload.html]: This line was too long for me. When I have to scroll horizontally I prefer to cut

**Comment by vied12** [comment]
`usageterms` has been duplicated (see https://github.com/superdesk/superdesk-client-core/pull/143#issuecomment-196367273)

File: scripts/superdesk-archive/views/upload.html (hunk 1 above)
Comment (vied12) [inline scripts/superdesk-archive/views/upload.html]: I forgot that I already cut the sentence... Updated for your pleasure

**Comment by naomiaro** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**